readthedocs/projects/templatetags/projects_tags.py

```diff
@@ -24,3 +24,8 @@ def sort_version_aware(versions):
 def is_project_user(user, project):
     """Return if user is a member of project.users."""
     return user in project.users.all()
+
+@register.assignment_tag
+def get_project_admin_url(project_pk):
+    """Returns the admin url for the given project's pk."""
```

Comment:
Isn't is possible to use the `url` tag using the URL named by the admin app instead of creating a tag here?

Reply:
Yes.
I have pushed the changes.